servicemesh-release-notes/topics/rn-fixed-issues.adoc

```diff
@@ -8,14 +8,18 @@ Fix - What did we change to fix the problem?
 Result - How has the behavior changed as a result?  Try to avoid “It is fixed” or “The issue is resolved” or “The error no longer presents”.
 ////
 
-https://github.com/Maistra/openshift-ansible/pull/19/[MASITRA-21] -  The default in the installer of 512Mi was too low for tracing.  Increased default Elasticsearch memory from 512 MB to 1GB.
-
 https://issues.jboss.org/browse/MAISTRA-4[MAISTRA-4] - The uninstall does not remove all the files, and as a result, when you re-install the istio-operator installation fails because `customresourcedefinitions.apiextensions.k8s.io "installations.istio.openshift.com"` already exists.
 
 https://issues.jboss.org/browse/MAISTRA-5[MAISTRA-5] - `openshift-ansible-istio-installer-job` pod tries to start but with errors.
 Graph layout - The layout for the Kiali graph can render differently, depending on your application architecture and the data to display (number of graph nodes and their interactions).  Because it is difficult if not impossible to create a single layout that renders nicely for every situation, Kiali offers a choice of several different layouts.  To choose a different layout, you can choose a different *Layout Schema* from the *Graph Settings* menu.
 
 https://issues.jboss.org/browse/MAISTRA-13[MAISTRA-13] - Installer should determine release version from installed components.  At present, the installer queries the yaml file, however if the yaml has been modified, the installer is not able to remove an older version.
 
+https://issues.jboss.org/browse/MAISTRA-79[MAISTRA-79] - Running the `istiooc cluster up` command results in the istio-operator namespace deploying a pod responsible for continually ensuring the Elastisearch sysctl requirements are met. This loop runs constantly causing a significant load on the system running the cluster.
+
+https://github.com/Maistra/openshift-ansible/pull/19/[MASITRA-21] -  The default in the installer of 512Mi was too low for tracing.  Increased default Elasticsearch memory from 512 MB to 1 GB.
```

Comment:
MAISTRA-21 and not MASITRA-21.  This seems to have been an old misspelling copied down to here.

Reply:
@knrc I fixed this.